### PR TITLE
Social media

### PR DESCRIPTION
**Pull request by ymaltsman**

### CODE REVIEW

**Comment by ymaltsman** [comment]
@b-meson just pushed edits that I think touched on all of your comments, let me know your thoughts. I also added an image

File: index.html [hunk 1]
```diff
@@ -369,24 +370,80 @@ <h1>Social Media Monitoring</h1>
   <div class="content">
   	<div class="row">
     <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
-    <p>Currently used by <a href="http://directives.chicagopolice.org/directives/data/a7a57b38-147b1793-bcd14-7b19-60f8232acf57b5f8.html">CPD</a>.</p>
+    <p>It is unclear where exactly <a href="http://directives.chicagopolice.org/directives/data/a7a57b38-147b1793-bcd14-7b19-60f8232acf57b5f8.html">CPD</a> has applied social media monitoring. However, CPD attempts to use social media monitoring to disrupt <a href="https://www.nbcchicago.com/news/local/aclu-raises-questions-about-chicago-police-social-media-surveillance/2749/">gang and gun violence</a>, so the practice likely disproportionately targets communities of color.</p>
```
Comment (b-meson) [inline index.html:373]: This is a small nit but I don't think we should state that CPD is using SOCMINT to disrupt gang violence. They _claim_ that as the purpose of social media monitoring (and therefore why its legitimate) but we know they also target protests and other activities.

File: index.html [hunk 2]
```diff
@@ -369,24 +370,80 @@ <h1>Social Media Monitoring</h1>
...
+	<p>CPD has also worked with <a href="https://www.propublica.org/article/chicago-public-schools-social-media-monitoring-violence-gangs">Chicago Public Schools</a> to implement social media monitoring in at least 24 schools, most of which are in Chicago’s South and West sides, to attempt to identify and intervene with students involved in gang and criminal activity. </p>
```
Comment (b-meson) [inline index.html:374]: same comment as above.

File: index.html [hunk 3]
```diff
@@ -369,24 +370,80 @@ <h1>Social Media Monitoring</h1>
...
+    <p>Invoices obtained through FOIA indicate that CPD spent <a href="https://onezero.medium.com/chicago-cops-use-social-media-to-track-grieving-families-of-gunshot-victims-e68e5a6dc40c">$1.25 million</a> on Dunami, a social media analytics software funded by <a href="https://en.wikipedia.org/wiki/In-Q-Tel">In-Q-Tel</a>, the venture capital arm of the CIA. </p>
+	<p>CPD has also spent <a href="https://www.documentcloud.org/documents/6567437-ACLU-v-CPD-Invoices.html">hundreds of thousands of dollars</a> for other social media related products and services over the past few years.</p>
+	<p>Chicago Public Schools’ social media monitoring program was funded by a <a href="https://www.ncjrs.gov/pdffiles1/nij/252491.pdf">$2.2 million grant</a> from the National Institute of Justice, which lasted from 2014 to 2018.</p>
+	</div>
+	<div class="row">
+    	<div class="col-md-9"></div>
+        <div class="col-md-3"><img src="img/OSINTlog.png" alt="Socmint" class="img-thumbnail"><h5>An OSINT log including social media posts, obtained by the ACLU via <a href="https://www.documentcloud.org/documents/6567425-Aclu-1-Reviewed-Redacted.html">FOIA.</a></h5></div>
```
Comment (b-meson) [inline index.html:384]: Can this image be centered? It looks a bit weird here.

File: index.html [hunk 4]
```diff
@@ -369,24 +370,80 @@ <h1>Social Media Monitoring</h1>
...
+	<h3> <span class="glyphicon glyphicon-wrench"></span> Efficacy </h3>
+    <p>CPD’s <a href="http://directives.chicagopolice.org/directives/">social media policy</a> claims that social media is a valuable tool for investigating crimes and gang activity, although the efficacy of social media monitoring is unclear.</p>
+	<p>The University of Chicago Crime Lab, which helped implement CPS’ social media monitoring program, <a href="https://www.wbez.org/stories/chicago-schools-monitoring-student-social-media-see-violence-drop/d0fbe47d-7a18-44b9-a276-38ca930833db">found</a> that schools which used social media monitoring experienced greater drops in misconduct and suspensions than those that didn't, although it did not find a significant difference in incidents of gun violence between these two sets of schools.</p>
```
Comment (b-meson) [inline index.html:393]: I am not sure if found is the proper term here. individual feedback for that on slack.

**Comment by b-meson** [comment]
Merge conflict is resolved, and closed via #20

**Pull request closed without merging**